src/uucore/src/lib/features/checksum.rs

```diff
@@ -462,6 +462,8 @@ pub fn perform_checksum_validation<'a, I>(
 where
     I: Iterator<Item = &'a OsStr>,
 {
+    let mut encountered_errors = false;
```

Comment:
I think this variable is not necessary and you could use `set_exit_code(1)` instead.

Reply:
smart, thanks!